packages/ui-new/src/composites/wui-list-select-account/index.ts

```diff
@@ -0,0 +1,74 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import '../../components/wui-text/index.js'
+import '../../components/wui-icon/index.js'
+import '../wui-avatar/index.js'
+import { UiHelperUtil } from '../../utils/UiHelperUtil.js'
+import type { IconType } from '../../utils/TypeUtil.js'
+
+@customElement('wui-list-select-account')
+export class WuiListSelectAccount extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() dollars = '0'
+
+  @property() pennies = '00'
+
+  @property() address = ''
```

Comment:
Same for the comment above, I think better practice instead of getting these values separately, getting the value and format it here with floating numbers. Also for such components it's important to make sure we're testing them with different values like very small values and very big numbers